internal/status/status.proto

```diff
@@ -37,6 +37,7 @@ message Component {
   string name = 1;              // component name (e.g., Cache)
   repeated int64 pids = 3;      // PIDs of component replicas
   repeated Method methods = 4;  // methods
+  map<int64, string> weaveletId = 5;
```

Comment:
Yes sure that can be done I am okay to add a new replica field, i added a new field because its a non-breaking change to the previously defined proto API and I was not sure if okay to do breaking changes.

If you are okay with it then sure I can add a new struct :)

Reply:
Good point. I think the only thing that can break is the `ssh` deployer, although I am not aware of anyone using it in prod.

I think we can add the new struct.

Reply:
Cool thanks.